eo-maven-plugin/src/test/java/org/eolang/maven/optimization/OptCachedTest.java

```diff
@@ -162,6 +196,19 @@ private static Path save(final Path tmp, final XML xml) throws IOException {
         return res;
     }
 
+    /**
+     * Set time of XML program use absolute path of program.
+     * @param path Temporary test directory.
+     * @param time XML program.
+     * @return Path to saved program.
+     */
+    private static void setTime(
```

Comment:
@Yanich96 Can we combine saving file and setting `lastModifierTime` in a single method? Seems, they usually go altogether.
Moreover, the name of the method is a bit confusing. Please, take a look this posts:
- https://www.yegor256.com/2018/08/22/builders-and-manipulators.html
- https://www.yegor256.com/2015/01/12/compound-name-is-code-smell.html

Reply:
@volodya-lombrozo I can't combine saving file and setting lastModifierTime in a single method, because method save() gets relative path and method setLastModifiedTime() gets absolutely path

Reply:
@Yanich96 The method `save` returns absolute path on that you can set last modification time